CODEOWNERS

```diff
@@ -27,42 +27,6 @@
 /bgp/ @cilium/sig-bgp
 /cmd/ @cilium/cli
 /clustermesh/ @cilium/sig-clustermesh
-/connectivity/ @cilium/ci-structure
-/connectivity/check/ipcache.go @cilium/ipcache
-/connectivity/check/metrics*.go @cilium/metrics
-/connectivity/check/policy.go @cilium/sig-policy
-/connectivity/builder/** @cilium/ci-structure
-/connectivity/builder/all_ingress_deny_from_outside.go @cilium/sig-encryption
-/connectivity/builder/cluster_entity_multi_cluster.go @cilium/sig-clustermesh
-/connectivity/builder/dns_only.go @cilium/sig-clustermesh
-/connectivity/builder/echo_ingress.go @cilium/sig-servicemesh
-/connectivity/builder/echo_ingress_auth_always_fail.go @cilium/sig-servicemesh
-/connectivity/builder/echo_ingress_from_other_client_deny.go @cilium/sig-servicemesh
-/connectivity/builder/echo_ingress_from_outside.go @cilium/sig-servicemesh
-/connectivity/builder/echo_ingress_knp.go @cilium/sig-servicemesh
-/connectivity/builder/echo_ingress_l7.go @cilium/sig-servicemesh
-/connectivity/builder/echo_ingress_l7_named_port.go @cilium/sig-servicemesh
-/connectivity/builder/echo_ingress_mutual_auth_spiffe.go @cilium/sig-servicemesh
-/connectivity/builder/egress_gateway.go @cilium/egress-gateway
-/connectivity/builder/egress_gateway_excluded_cidrs.go @cilium/egress-gateway
-/connectivity/builder/no_ipsec_xfrm_errors.go @cilium/sig-encryption
-/connectivity/builder/node_to_node_encryption.go @cilium/sig-encryption
-/connectivity/builder/pod_to_pod_encryption.go @cilium/sig-encryption
-/connectivity/tests/egressgateway.go @cilium/egress-gateway
-/connectivity/tests/encryption.go @cilium/sig-encryption
-/connectivity/tests/errors.go @cilium/sig-agent @cilium/sig-datapath
-/connectivity/tests/externalworkload.go @cilium/sig-clustermesh
-/connectivity/tests/from-cidr.go @cilium/sig-policy
-/connectivity/tests/health.go @cilium/sig-agent
-/connectivity/tests/host.go @cilium/sig-agent
-/connectivity/tests/ipsec_xfrm.go @cilium/ipsec
-/connectivity/tests/perfpod.go @cilium/sig-datapath
-/connectivity/tests/pod.go @cilium/sig-agent
-/connectivity/tests/service.go @cilium/sig-lb
-/connectivity/tests/testloop.sh @jrajahalme
-/connectivity/tests/to-cidr.go @cilium/sig-policy
-/connectivity/tests/upgrade.go @cilium/sig-datapath
-/connectivity/tests/world.go @cilium/proxy
 /encrypt/ @cilium/sig-encryption
```

Comment:
I don't see these codeowners being migrated over in [cilium/cilium#](https://github.com/cilium/cilium/pull/31077/files#diff-fcf14c4b7b34fe7a11916195871ae66a59be87a395f28db73e345ebdc828085b). I think teams should probably keep owning connectivity tests for features they own.

Reply:
moved all the code owners over to https://github.com/cilium/cilium/pull/31077 except for:
```
/connectivity/tests/testloop.sh @jrajahalme
```
i think we added this line as a joke at some point 😅